src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -70,6 +72,11 @@
   @Description("Type of write operation to perform. This can be set to Insert, Update or Upsert.")
   protected String operation;
 
+  @Name(NAME_WRITE_DISPOSITION)
+  @Macro
+  @Description("Type of write disposition to use. This can be set to Empty, Truncate, or Append.")
+  protected String writeDisposition;
```

Comment:
this should also be annotated as Nullable. If null, it should default to false. 

It won't ever be null if the pipeline is created through the UI, but it can be if somebody creates it through the REST endpoint.

Reply:
It won't be nullable if it uses the primitive boolean rather than the Boolean wrapper class, right?

Reply:
should change it to a Boolean. The nullable annotation also tells CDAP that the user doesn't have to provide this property. If it's not annotated as nullable, CDAP will complain if the user doesn't provide it.

Reply:
Gotcha, fixed